.github/ISSUE_TEMPLATE/functional_bug.md

```diff
@@ -9,55 +9,41 @@ assignees: ''
 
 # Summary
 
-Replace this with a short summary (one sentence or short paragraph) of
-the issue you are describing.
+Replace this with a short summary (one sentence or short paragraph) of the issue you are describing.
 
 ## Affected paths
 
-List the URL(s) where you found inaccurate information. You can use
-URLs from [the rendered site](https://docs.cleura.cloud) or from [the
-sources on GitHub](https://github.com/citynetwork/docs), either is
-fine.
+List the URL(s) where you found inaccurate information. You can use URLs from [the rendered site](https://docs.cleura.cloud) or from [the sources on GitHub](https://github.com/citynetwork/docs), either is fine.
 
 ## What I did
 
-Explain the steps you undertook that produced results that are
-inconsistent with the documentation, or that led to errors when you
-followed the documentation.
+Explain the steps you undertook that produced results that are inconsistent with the documentation, or that led to errors when you followed the documentation.
 
-It’s OK to remove or mask credentials, but otherwise try to err on the
-side of sharing more rather than less information.
+It’s OK to remove or mask credentials, but otherwise try to err on the side of sharing more rather than less information.
```

Comment:
Not super important, but a simpler/more common syntax would probably be better here. So, instead of "...try to err on the side of sharing...", I would say "...it's preferable to share...".

Reply:
Yeah, not now. Please approve as-is and then we can fix that in a later PR.